core/monitor/monitor.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/golang/glog"
 
 	"github.com/google/keytransparency/core/monitor/storage"
-	ktpb "github.com/google/keytransparency/core/proto/keytransparency_v1_types"
+	ktpb "github.com/google/keytransparency/core/proto/keytransparency_v1"
```

Comment:
nit: use pb package alias, as you've done elsewhere
same for other monitor files.

Reply:
done